docs/examples/Configuring_QCoDeS.ipynb

```diff
@@ -186,8 +187,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This will add a base_location with value /dev/random to the current config, and validate it's value to be of type string, will also set the description and what one would want to have as default.\n",
-    "The new entry is saved in the 'user' part of the config."
+    "This will add a base_location with value /dev/random to the current configuration, and validate it's value to be of type string, will also set the description and what one would want to have as default.\n",
```

Comment:
since we're improving the documentation, i guess the formatting of text can be improved on the way, right?
```suggestion
    "This will add a `base_location` with value `/dev/random` to the current configuration, and validate it's value to be of type string, will also set the description and what one would want to have as default.\n",
```